### PR TITLE
coq-elpi 1.4 for Coq 8.11

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: released/packages/coq-elpi/coq-elpi.1.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage: "https://github.com/LPCIC/coq-elpi"
+bug-reports: "https://github.com/LPCIC/coq-elpi/issues"
+dev-repo: "git+https://github.com/LPCIC/coq-elpi"
+
+build: [ make "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ]
+install: [ make "install" "COQBIN=%{bin}%/" "ELPIDIR=%{prefix}%/lib/elpi" ]
+depends: [
+  "elpi" {>= "1.11.2" & < "1.12.0~"}
+  "coq" {>= "8.11" & < "8.12~"}
+  ]
```
Comment (clarus) [inline released/packages/coq-elpi/coq-elpi.1.4.0/opam:14]:
```suggestion
  ]
  tags: [
    "logpath:elpi"
  ]
```

**Comment by clarus** [comment]
You seem to have the same build error as on my PR https://github.com/coq/opam-coq-archive/pull/1205 It seems to me that `dune` attempts to read itself as a Dune file.

**Comment by gares** [comment]
I've no idea, I've never seen this error before.
Here a failing log https://gitlab.com/coq/opam-coq-archive/-/jobs/559225324
CC @ejgallego @Zimmi48

**Comment by gares** [comment]
It must be related to the setup here, since I've installed elpi via opam earlier this afternoon using the same dune version, and it worked.

**Comment by gares** [comment]
On my machine:
```
gares@ollypat:~$ opam switch create 4.09.0

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[ocaml-base-compiler.4.09.0] downloaded from cache at https://opam.ocaml.org/cache

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
∗ installed base-bigarray.base
∗ installed base-threads.base
∗ installed base-unix.base
∗ installed ocaml-base-compiler.4.09.0
∗ installed ocaml-config.1
∗ installed ocaml.4.09.0
Done.
# Run eval $(opam env) to update the current shell environment
gares@ollypat:~$ opam switch create 4.09.0^C
gares@ollypat:~$ eval $(opam env)
gares@ollypat:~$ opam install elpi
The following actions will be performed:
  ∗ install dune                    2.5.1      [required by elpi]
  ∗ install seq                     base       [required by re]
  ∗ install camlp5                  7.11       [required by elpi]
  ∗ install conf-m4                 1          [required by ocamlfind]
  ∗ install sexplib0                v0.13.0    [required by base]
  ∗ install result                  1.5        [required by
                                               ocaml-migrate-parsetree,
                                               ppx_deriving]
  ∗ install ppx_tools               6.0+4.08.0 [required by ppx_deriving]
  ∗ install ppx_derivers            1.2.1      [required by ppx_deriving,
                                               ocaml-migrate-parsetree,
                                               ppxlib]
  ∗ install ocaml-compiler-libs     v0.12.1    [required by ppxlib]
  ∗ install dune-private-libs       2.5.1      [required by dune-configurator]
  ∗ install cppo                    1.6.6      [required by ppx_deriving]
  ∗ install re                      1.9.0      [required by elpi]
  ∗ install ocamlfind               1.8.1      [required by ppxfind]
  ∗ install ocaml-migrate-parsetree 1.7.3      [required by elpi]
  ∗ install dune-configurator       2.5.1      [required by base]
  ∗ install ppxfind                 1.4        [required by ppx_deriving]
  ∗ install base                    v0.13.2    [required by ppxlib]
  ∗ install ppx_deriving            4.4.1      [required by elpi]
  ∗ install stdio                   v0.13.0    [required by ppxlib]
  ∗ install ppxlib                  0.13.0     [required by elpi]
  ∗ install elpi                    1.11.2
===== ∗ 21 =====
Do you want to continue? [Y/n] y

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[camlp5.7.11] found in cache
[cppo.1.6.6] found in cache
[dune.2.5.1] found in cache
[dune-configurator.2.5.1] found in cache
[dune-private-libs.2.5.1] found in cache
[elpi.1.11.2] found in cache
[ocaml-compiler-libs.v0.12.1] found in cache
[ocaml-migrate-parsetree.1.7.3] found in cache
[base.v0.13.2] downloaded from cache at https://opam.ocaml.org/cache
[ocamlfind.1.8.1] found in cache
[ppx_derivers.1.2.1] found in cache
[ppx_deriving.4.4.1] found in cache
[ppx_tools.6.0+4.08.0] found in cache
[ppxfind.1.4] found in cache
[ppxlib.0.13.0] found in cache
[re.1.9.0] found in cache
[result.1.5] found in cache
[sexplib0.v0.13.0] found in cache
[stdio.v0.13.0] found in cache

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
∗ installed seq.base
∗ installed conf-m4.1
∗ installed ocamlfind.1.8.1
∗ installed dune.2.5.1
∗ installed cppo.1.6.6
∗ installed ppx_derivers.1.2.1
∗ installed result.1.5
∗ installed ocaml-compiler-libs.v0.12.1
∗ installed ppx_tools.6.0+4.08.0
∗ installed dune-private-libs.2.5.1
∗ installed re.1.9.0
∗ installed sexplib0.v0.13.0
∗ installed ocaml-migrate-parsetree.1.7.3
∗ installed dune-configurator.2.5.1
∗ installed ppxfind.1.4
∗ installed ppx_deriving.4.4.1
∗ installed base.v0.13.2
∗ installed camlp5.7.11
∗ installed stdio.v0.13.0
∗ installed ppxlib.0.13.0
∗ installed elpi.1.11.2
Done.
gares@ollypat:~$ 
```
So I believe the error has something to do with this CI

**Comment by gares** [comment]
@clarus shall we ignore the failure? it must be about dune and --unlock-base, since it works on the same compiler version, both here and in CI

**Comment by gares** [comment]
I tried locally with the --unlock-base flag on an empty 4.09 switch, and it works.
I'm ignoring the job opam-build:any and merge

**Comment by clarus** [comment]
> @clarus shall we ignore the failure? it must be about dune and --unlock-base, since it works on the same compiler version, both here and in CI

OK for me!

**Pull request merged**